### ISSUE TITLE
copy_tinytex without being a R user

### ISSUE THREAD

**Issue by mateoc10** [yihui/tinytex#89]
Hello,
Is it posible to use the copy_tinytex and use_tinytex without being a R user?
Great work with TinyTeX, its really helpful!
Thanks a lot!

**Comment by yihui** [comment]
Yes, but only if someone can help translate my R code to shell scripts:
https://github.com/yihui/tinytex/blob/eacaaf11747868b8229489b5620e88fb8a9d85d3/R/install.R#L295-L335

Essentially, `copy_tinytex()` is pretty much `cp -r source dest`, and `use_tinytex()` runs `the/new/full/path/to/tlmgr path add`.
